0.3.21 does not load: Cannot find module 'react'

## Actual Behavior

After the update to 0.3.21, the following is logged to "Output > Log (Extension Host)" when VS Code starts:

```
[2021-11-24 15:20:25.442] [exthost] [error] Activating extension GraphQL.vscode-graphql failed due to an error:
[2021-11-24 15:20:25.442] [exthost] [error] Error: Cannot find module 'react'
Require stack:
- /Users/xxx/.vscode/extensions/graphql.vscode-graphql-0.3.21/out/extension.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/loader.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/bootstrap-amd.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/bootstrap-fork.js
```

## Specifications

  - GraphQL for VSCode Extension Version: 0.3.21
  - VSCode Version: 1.62.3
  - OS Name: MacOS
  - OS Version: 12.0.1
  - graphql config filename and format example:

Thanks for looking into it!

--- a/package.json
+++ b/package.json
@@ -208,8 +208,8 @@
   },
   "scripts": {
     "vscode:prepublish": "npm run compile -- --minify",
-    "compile": "npm run compile:server && esbuild ./src/extension.ts --bundle --outfile=out/extension.js --external:vscode --external:react --format=cjs --platform=node",
-    "compile:server": "esbuild ./src/server/server.ts --bundle --outfile=out/server/server.js --external:vscode --external:react --format=cjs --platform=node",
+    "compile": "npm run compile:server && esbuild ./src/extension.ts --bundle --outfile=out/extension.js --external:vscode --format=cjs --platform=node",
+    "compile:server": "esbuild ./src/server/server.ts --bundle --outfile=out/server/server.js --external:vscode --format=cjs --platform=node",
     "build": "npm run compile -- --sourcemap",
     "watch": "npm run build --watch",
     "postinstall": "node ./node_modules/vscode/bin/install",
@@ -238,6 +238,7 @@
     "@graphql-tools/load": "^7.4.1",
     "@graphql-tools/url-loader": "^7.5.2",
     "@graphql-tools/wrap": "^8.3.2",
+    "@urql/core": "^2.3.5",
     "babel-polyfill": "6.26.0",
     "capitalize": "^2.0.4",
     "dotenv": "^10.0.0",
@@ -247,7 +248,6 @@
     "graphql-tag": "^2.12.6",
     "graphql-ws": "^5.5.5",
     "node-fetch": "^2.6.6",
-    "urql": "^2.0.5",
     "vscode-languageclient": "^5.2.1",
     "ws": "^8.2.3"
   },

--- a/src/client/network-helper.ts
+++ b/src/client/network-helper.ts
@@ -13,7 +13,7 @@ import {
   createClient,
   defaultExchanges,
   subscriptionExchange,
-} from "urql"
+} from "@urql/core"
 
 import {
   ExtractedTemplateLiteral,

